### PR TITLE
Add language for Maithili, Bodo, Goan Konkani

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Link : -

### Problem Statement

There are some missing language in our language list 

### Note

Maithili : मैथिली
Bodo : बड़ो
Goan Konkani (Devanagari script) : गोवा कोंकणी

### CODE REVIEW

**Comment by hueitan** [comment]
@jpita can you confirm this with the language team, so to make sure we don't add the wrong name or display name to the language?

File: src/utils/languages.js [hunk 1]
```diff
@@ -1440,6 +1440,21 @@ const languages = [
     canonical_name: 'Herero',
     code: 'hz',
     name: 'Otsiherero'
+  },
+  {
+    canonical_name: 'Maithili',
+    code: 'mai',
+    name: '\u092E\u0948\u0925\u093F\u0932\u0940'
```
Comment (Nikerabbit) [inline src/utils/languages.js]: Is there a specific need to use unicode escapes? Most JSON parsers can deal with Unicode.

Comment (hueitan) [reply src/utils/languages.js]: not really, the initial list is given in the Unicode escape form, therefore we are doing the same here.

**Comment by Abijeet** [comment]
The language codes used for the languages look fine to me.

**Pull request merged**